src/Discord.Net.Commands/CommandServiceConfig.cs

```diff
@@ -38,7 +38,56 @@ public class CommandServiceConfig
             {'﹃', '﹄' },
             {'＂', '＂' },
             {'＇', '＇' },
-            {'｢', '｣' }
+            {'｢', '｣' },
+            {'(', ')' },
+            {'༺', '༻' },
+            {'༼', '༽' },
+            {'᚛', '᚜' },
+            {'⁅', '⁆' },
+            {'⌈', '⌉' },
+            {'⌊', '⌋' },
+            {'❨', '❩' },
+            {'❪', '❫' },
+            {'❬', '❭' },
+            {'❮', '❯' },
+            {'❰', '❱' },
+            {'❲', '❳' },
+            {'❴', '❵' },
+            {'⟅', '⟆' },
+            {'⟦', '⟧' },
+            {'⟨', '⟩' },
+            {'⟪', '⟫' },
+            {'⟬', '⟭' },
+            {'⟮', '⟯' },
+            {'⦃', '⦄' },
+            {'⦅', '⦆' },
+            {'⦇', '⦈' },
+            {'⦉', '⦊' },
+            {'⦋', '⦌' },
+            {'⦍', '⦎' },
+            {'⦏', '⦐' },
+            {'⦑', '⦒' },
+            {'⦓', '⦔' },
+            {'⦕', '⦖' },
+            {'⦗', '⦘' },
+            {'⧘', '⧙' },
+            {'⧚', '⧛' },
+            {'⧼', '⧽' },
+            {'⸂', '⸃' },
+            {'⸄', '⸅' },
+            {'⸉', '⸊' },
+            {'⸌', '⸍' },
+            {'⸜', '⸝' },
+            {'⸠', '⸡' },
+            {'⸢', '⸣' },
+            {'⸤', '⸥' },
+            {'⸦', '⸧' },
+            {'⸨', '⸩' },
+            {'【', '】'},
+            {'〔', '〕' },
+            {'〖', '〗' },
+            {'〘', '〙' },
+            {'〚', '〛' }
```

Comment:
IMO if we're adding a large dict of things like this, it might be better to use a separate file for it instead. I'm not sure though.

Reply:
Do you mean deserializing the dictionary from another file, or another file in the source that defines the default value of this dictionary?

I think that including all of these by default is a good idea. Alternatively we could shorten this list and include a more complete one in the documentation.

Reply:
It looks like that gist uses features that aren't supported in .netstandard1.1 https://docs.microsoft.com/en-us/dotnet/api/system.char.getunicodecategory?view=netcore-2.0#Applies_to 
https://docs.microsoft.com/en-us/dotnet/api/system.string.normalize?view=netcore-2.0#Applies_to

So for now it may be best to define this dictionary in another file and leave it.